packages/sockethub/bin/sockethub

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env node
+const SockethubServer = require('@sockethub/server');
+SockethubServer();
```

Comment:
This looks a bit weird to me. Usually, a capitalized variable should be a class/constructor, not a function that starts a server.